Person.h

```diff
@@ -0,0 +1,14 @@
+ #import <Foundation/Foundation.h>
+ #import "Organization.h"
+@interface Person:NSObject{
+    NSString *firstName;
+    NSString *lastName;
+    NSNumber *age;
+}
+@property NSString *firstName;
+@property NSString *lastName;
+@property NSNumber *age;
+-(id) initWithPersonDetails: (NSString *) fn andLastName: (NSString*) ln andAge: (NSNumber*) a;
```

Comment:
> -(id) initWithPersonDetails: (NSString *) fn andLastName: (NSString*) ln andAge: (NSNumber*) a;
Remove unrequired spaces and don't use abbreviations for parameters.
This should be
> -(id)initWithPersonDetails:(NSString *)firstName andLastName:(NSString*)lastName andAge:(NSNumber*)age;